src/query/api/experimental/annotated/handler.go

```diff
@@ -52,7 +52,7 @@ var errEmptyBody = errors.New("request body is empty")
 type Handler struct {
 	writer     ingest.DownsamplerAndWriter
 	tagOptions models.TagOptions
-	metrics    handlerMetrics
+	//metrics    handlerMetrics
```

Comment:
All these changes in handler.go are temporary just to push things forward and make it work. Jerome is working on it permanent solution.